en/developer_guide/media_standalone_display.rst

```diff
@@ -0,0 +1,182 @@
+Medias display
+==============
+
+Context
+-------
+
+To improve your website performances, a common way is to download the page datas from
+multiple servers. In fact, in the HTTP protocol, a browser can make only two parallel
+request to the same url. Recent browser (such as ``Chrome`` or ``Firefox`` can perform
```

Comment:
not clear when you speak about two request after. Why not ? 
In fact, browser can make a restraint number of parallel request to the same url.